basedtyping/__init__.py

```diff
@@ -231,9 +240,10 @@ def issubform(form: type | UnionType, forminfo: type | UnionType) -> bool:
     >>> issubform(int | str, object)
     True
     """
+    # type ignores because issubclass doesn't support _SpecialForm but we do
```

Comment:
Should it be cast instead?

Reply:
i dont think so because they aren't `type`s